feelpp/feel/feeldiscr/createsubmesh.hpp

```diff
@@ -393,7 +400,8 @@ CreateSubmeshTool<MeshType,IteratorRange,TheTag>::build( mpl::int_<MESH_ELEMENTS
             } // for (unsigned int n=0 ... )
 
             // Add an equivalent element type to the new_mesh
-            auto const& e = newMesh->addElement( newElem,true );
+            auto eit = newMesh->addElement( newElem,true );
```

Comment:
would it be possible to extract it here the iterator and the boolean ?

Reply:
we would have
```c++
auto [eit,inserted] = newMesh->addElement( newElem,true );
auto const& [key,e] = *eit;
```

Reply:
perhaps a good regexp could automate this?
all these lines are pretty much the same